samples/drivers/eeprom/sample.yaml

```diff
@@ -11,9 +11,20 @@ common:
 tests:
   sample.drivers.eeprom:
     tags: eeprom
-    platform_allow: native_posix gd32f450i_eval
+    platform_allow: >
+      gd32f450i_eval
+      native_posix
+      native_posix_64
     integration_platforms:
-      - native_posix
+      - native_posix_64
+  sample.drivers.eeprom.shell:
+    tags: eeprom
+    platform_allow: >
+      gd32f450i_eval
+      native_posix
+      native_posix_64
+    extra_args: OVERLAY_CONFIG=overlay-shell.conf
+    harness: keyboard
```

Comment:
You could combine the common properties in a `common` section.

Reply:
Tried to improve, but could not really find any properties which could be moved to common. Which ones are you thinking of?